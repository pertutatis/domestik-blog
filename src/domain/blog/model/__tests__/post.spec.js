import Post from '../post'
import mockedPost from './__mocks__/post'

describe('Post Model', () => {
  it('The model maintains the structure', () => {
    const post = new Post(mockedPost)

    expect(post).toMatchSnapshot()
  })
})
