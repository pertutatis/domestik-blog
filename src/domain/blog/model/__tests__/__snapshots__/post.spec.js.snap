// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post Model The model maintains the structure 1`] = `
Post {
  "body": "est rerum tempore vitae
sequi sint nihil reprehenderit dolor beatae ea dolores neque
fugiat blanditiis voluptate porro vel nihil molestiae ut reiciendis
qui aperiam non debitis possimus qui neque nisi nulla",
  "date": "01-02-2021",
  "postId": undefined,
  "title": "qui est esse",
  "userId": 1,
}
`;
